gabarit/template_nlp/generate_nlp_project.py

```diff
@@ -21,12 +21,14 @@
 import argparse
 import tempfile
 import configparser
+import pkg_resources
 from typing import Union
 from shutil import copyfile, rmtree
 from distutils.dir_util import copy_tree
 from jinja2 import Environment, FileSystemLoader
 
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+GABARIT_VERSION = pkg_resources.get_distribution("gabarit").version
```

Comment:
So that mean that Gabarit must be "installed" ? That was not compulsory before.

Reply:
Added a try/catch